### PR TITLE
Fix #301, fit markdown web link styler

### LINKED ISSUE #301: MarkdownStyler should render links in blue
https://github.com/yuanjian-org/app/commit/b113c2be93533df56e2bef6b7afe4971cbdd9546 enabled remark-gfm. However, URLs are not rendered in blue:

image
image

### PR DESCRIPTION
**Pull request by UncleBu1728**
Fix #301, `MarkDownStyler.css` has been updated to show links in[ blue and an underline](https://www.youtube.com/watch?v=dQw4w9WgXcQ&ab_channel=RickAstley) when the cursor is on the link.  

image
image

### CODE REVIEW

File: src/theme/MarkdownStyler.module.css [hunk 1]
```diff
@@ -26,3 +26,11 @@
 .markdownStyler p {
   margin: 0.75em 0;
 }
+
+.markdownStyler a {
+  color: blue;
+}
+
+.markdownStyler a:hover {
```
Comment (weihanwang) [inline src/theme/MarkdownStyler.module.css:34]: no need for the underline. Standard Chakra links don't have underlines, either.

File: src/theme/MarkdownStyler.module.css [hunk 2]
```diff
@@ -26,3 +26,11 @@
 .markdownStyler p {
   margin: 0.75em 0;
 }
+
+.markdownStyler a {
+  color: blue;
```
Comment (weihanwang) [inline src/theme/MarkdownStyler.module.css:31]: Can you find out the color styling that standard Chakra links use?

Comment (weihanwang) [reply src/theme/MarkdownStyler.module.css:31]: Check out theme/links.ts

Comment (weihanwang) [reply src/theme/MarkdownStyler.module.css:31]: It would be ideal if we could use chakra class name here.

Comment (UncleBu1728) [reply src/theme/MarkdownStyler.module.css]: I have picked the color teal.500 due to the [chakra document here ](https://v2.chakra-ui.com/docs/components/link#link-inline-with-text)

**Comment by UncleBu1728** [comment]
the latest screenshot of links,

image

comparing with default Chakra UI link color

image

File: src/theme/MarkdownStyler.module.css [hunk 3]
```diff
@@ -28,9 +28,5 @@
 }
 
 .markdownStyler a {
-  color: blue;
-}
-
-.markdownStyler a:hover {
-  text-decoration: underline;
+  color: #319795;
```
Comment (weihanwang) [inline src/theme/MarkdownStyler.module.css:31]: add a comment here saying this mirrors brand color and refer to the file name.

also add a comment in links.ts, saying when updating this color, search for all occurrences of this color and change them all.

and why is this color code different than brand.c, the the color specified in links.ts for links?

Comment (UncleBu1728) [reply src/theme/MarkdownStyler.module.css:31]: image
It matches the color.brand.c now

File: src/theme/links.ts [hunk 4]
```diff
@@ -4,6 +4,7 @@ const links = {
   components: {
     Link: {
       baseStyle: {
+        // If the link color(currently brand.c) is changed, search for all the occurances of the color used and update all instances
```
Comment (weihanwang) [inline src/theme/links.ts:7]: Change comment to "If changing this line, also update link color in MarkdownStyler.module.css"

**Review by weihanwang — APPROVED** [review]

**Pull request merged**